sdk/framework/backend.go

```diff
@@ -530,14 +530,9 @@ func (b *Backend) handleRootHelp(req *logical.Request) (*logical.Response, error
 		return nil, err
 	}
 
-	// Plugins currently don't have a direct knowledge of their own "type"
-	// (e.g. "kv", "cubbyhole"). It defaults to the name of the executable but
-	// can be overridden when the plugin is mounted. Since we need this type to
-	// form the request & response full names, we are passing it as an optional
-	// request parameter to the plugin's root help endpoint. If specified in
-	// the request, the type will be used as part of the request/response body
-	// names in the OAS document.
-	requestResponsePrefix := req.GetString("requestResponsePrefix")
+	// If specified in the request, the mount path will be used as part of the
+	// request/response body names and operation id's in the OpenAPI document.
+	defaultMountPath := req.GetString("defaultMountPath")
```

Comment:
Is the only plugin that will be affected here `kv` b/c the default mount path is `secret`? Or are there others?

Reply:
Gotcha! I misunderstood.